frontend/main.js

```diff
@@ -90,7 +91,7 @@ async function startApp () {
       return {
         ephemeral: {
           syncs: [],
-          finishedLogin: 'no'
+          finishedLogin: this.$store.state.loggedIn ? 'yes' : 'no'
```

Comment:
I think this is a potentially troublesome change, and would recommend keeping it the way it was...

Reply:
Hum... reverted but left a comment to review this later.